rpc/serde/jsonrpc.go

```diff
@@ -99,7 +109,9 @@ type ResponsePayload interface {
 		query.LedgerChannelInfo |
 		VersionResponse |
 		GetAllLedgersResponse |
-		GetPaymentChannelsByLedgerResponse
+		GetPaymentChannelsByLedgerResponse |
+		payments.Voucher |
+		uint64
```

Comment:
I used a `uint64` for convenience but this probably should be a `big.Int`

Reply:
https://github.com/statechannels/go-nitro/pull/1385/commits/5e799eb3694346f4ef47416d3b5bc2488fcd729f